### PR TITLE
fixed errors in documentation

### PR DESCRIPTION
**Pull request by piterand**

### CODE REVIEW

File: ammico/cropposts.py [hunk 1]
```diff
@@ -326,12 +326,14 @@ def crop_media_posts(
     # get the reference images with regions that signify areas to crop
     ref_views = []
     for ref_file in ref_files:
-        ref_view = cv2.imread(ref_file)
+        ref_file_path = ref_files[ref_file]["filename"]
```
Comment (iulusoy) [inline ammico/cropposts.py:329]: `ref_file["filename"]` should be enough

File: ammico/cropposts.py [hunk 2]
```diff
@@ -326,12 +326,14 @@ def crop_media_posts(
...
     for ref_file in ref_files:
-        ref_view = cv2.imread(ref_file)
+        ref_file_path = ref_files[ref_file]["filename"]
+        ref_view = cv2.imread(ref_file_path)
         ref_views.append(ref_view)
     # parse through the social media posts to be cropped
     for crop_file in files:
-        view = cv2.imread(crop_file)
-        print("Doing file {}".format(crop_file))
+        crop_file_path = files[crop_file]["filename"]
```
Comment (iulusoy) [inline ammico/cropposts.py:334]: same here, `crop_file["filename"]` should be sufficient

**Review by iulusoy — COMMENTED** [review]
True, we forgot to update cropposts after writing the `find_files` wrapper.. lgtm apart from those two small changes

File: ammico/cropposts.py (hunk 2 above)
Comment (piterand) [reply ammico/cropposts.py:334]: Oh..., yeah, sure 🤦.

**Pull request merged**